main.go

```diff
@@ -58,6 +62,7 @@ func run(ec2Service ec2iface.EC2API, autoScalingService autoscalingiface.AutoSca
 	cfg := config.Get()
 	client, err := k8s.CreateClientSet()
 	if err != nil {
+		metrics.Server.Errors.Inc()
 		return errors.New("unable to create Kubernetes client: " + err.Error())
 	}
```

Comment:
Can you move this `metrics.Server.Errors.Inc()` to the parent function instead? That way you won't have to do it both here, on line 80 and on line 105 of the same file.